.travis.yml

```diff
@@ -1,7 +1,10 @@
+# For Travis to run this, make sure to add a Webhook in the Github project
```

Comment:
Might be useful to spell out  Settings > Webhooks. Also, I think this is called repository, not project.